src/domain/flow-system/domain/src/entities/flow_configuration/flow_configuration_status.rs

```diff
@@ -12,16 +12,14 @@
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum FlowConfigurationStatus {
     Active,
-    PausedTemporarily,
-    StoppedPermanently,
+    Deleted,
 }
 
 impl FlowConfigurationStatus {
     pub fn is_active(&self) -> bool {
         match self {
             FlowConfigurationStatus::Active => true,
-            FlowConfigurationStatus::PausedTemporarily
-            | FlowConfigurationStatus::StoppedPermanently => false,
+            FlowConfigurationStatus::Deleted => false,
```

Comment:
1) We are not allowing multiple triggers at the moment. Doing so is another modeling surgery like this branch.
Anyway, when this is supported, multiple failures could initiate pausing all triggers that exist for the flow, and later can be re-enabled individually or altogether.

2) I don't understand the model when an individual flow can be paused. We have to pause the series of flows for the same flow key, rather than an individual run. If it's not the trigger's property, it's an entity we currently don't have in our model.

Reply:
Ok I see. The current `Flow` is more like a `FlowRun`. We'll need to separate the "what to do" (flow) from "how it was executed" (flow run) to achieve what I described. But whether it's worth doing is a separate conversation.